modules/gdnative/gdnative.h

```diff
@@ -43,7 +43,8 @@
 class GDNativeScriptData;
 class GDNativeLibrary;
 
-struct NativeLibrary {
+class NativeLibrary {
```

Comment:
It's not critical though, it's just weird. You don't need to change this if you don't want to ^^